### ISSUE TITLE
Skyrim Script Extender (SKSE)

### ISSUE THREAD

**Issue by Dragonisser** [tiltedphoques/Mod-Compatibility#2]
# Mod information

#### Download link

http://skse.silverlock.org/

#### Dependencies

- None

# Compatibility

> Mod is compatible, does nothing alone.

# Additional Information

> Mods based on SKSE with custom scripts wont sync.

**Comment by DrogonPrime** [comment]
Needed to have this installed to see the Together server meny (right-ctrl)

**Comment by Dragonissa** [comment]
> Needed to have this installed to see the Together server meny (right-ctrl)

We dont rely on SKSE so this makes barely any sense to me.

**Comment by DrogonPrime** [comment]
I agree with u but tried to open it without it dident work. Tried to restart the game and reload the save. That worked for some but not sure if they had SKSE or not.

**Comment by lbadi** [comment]
Hey folks, is there any extra step to take in order to launch reborn with SKSE enabled ? I've everything set up using vortex, but when i launch Together Reborn doesn´t open SKSE with him.

**Comment by lbadi** [comment]
> Hey folks, is there any extra step to take in order to launch reborn with SKSE enabled ? I've everything set up using vortex, but when i launch Together Reborn doesn´t open SKSE with him.

Because i had to downgrade the version of Skyrim because of [this](https://www.reddit.com/r/skyrimmods/comments/xetnls/comment/ioin7ls/) i had to install an older version of SKSE manually to make the trick.

**Comment by Sidistyk** [comment]
> > Hey folks, is there any extra step to take in order to launch reborn with SKSE enabled ? I've everything set up using vortex, but when i launch Together Reborn doesn´t open SKSE with him.
> 
> Because i had to downgrade the version of Skyrim because of [this](https://www.reddit.com/r/skyrimmods/comments/xetnls/comment/ioin7ls/) i had to install an older version of SKSE manually to make the trick.

So what version of this mod do we use? Hard to follow that reddit convo since I can't tell if it applies to me. Running SkyrimSE*

**Comment by Sidistyk** [comment]
So how do we get skse to load from skyrim together launcher? Do I point the launcher to the skse launcher, because that does not seem to work, it says "Failed to load Skyrim Address Lib" and tells me to get the All in one Anniversary Edition, which I have. When I load the game through the SkyrimSE launcher it does not show SKSE is enabled.

Thoughts?

**Comment by Pingu-Pox** [comment]
> So how do we get skse to load from skyrim together launcher? Do I point the launcher to the skse launcher, because that does not seem to work, it says "Failed to load Skyrim Address Lib" and tells me to get the All in one Anniversary Edition, which I have. When I load the game through the SkyrimSE launcher it does not show SKSE is enabled.
> 
> Thoughts?

Skyrim Together will automatically load SKSE if it is present, you are still launching SkyrimTogether.exe to accomplish this. SKSE is not required by STR, so please remove it and troubleshoot your issue with AddressLib (not to be confused with SKSE, they are not the same).

**Comment by kitrady** [comment]
hey, doesn't seem like it was mentioned here (or maybe I missed it), but on the discord sever for Skyrim Together Reborn, a bot for the server said that SKSE will cause the STRui to not always come up for unknown reasons, which feels like an important piece of info for people looking at this page, so be warned that you maybe constantly be fighting for the STRui to appear if you use SKSE with STR (for me it seems to be a problem for the non-party leaders, the STRui will only sometimes come up for them)

**Comment by Corleone91** [comment]
Greetings! how to fix it?

пн, 2 янв. 2023 г. в 23:25, Gender-Vortex ***@***.***>:

**Comment by arakan94** [comment]
I can't open STRui and connect to server when I am running SKSE (everything is latest version - SKSE is 2.2.3, installed via Vortex).

Any advice on how to fix or circumvent it? I don't think I can play Skyrim without SkyUI... :(

**Comment by kitrady** [comment]
Okay, there could be LOTS of problems, the STRui and the mod in general is very finicky in my experience, but if you sure you have everything installed right and followed the whole guide on the wiki and everything, my one idea that could possibly fix things is to run the STR exe directly from the directory. I know the wiki says to create a shortcut on vortex, but for some reason I’ve found the shortcut to cause the STRui to not come up, and the same thing with a desktop shortcut, I'm like 99% I’ve never had the STRui not show up if I launch directly from file explorer, but have had it not show up several times with other launching methods

**Comment by arakan94** [comment]
I had previously activated debug info (for ping) before installing SKSE, so I knew I had the correct exe running as the debug info still showed. What actually helped me was PC restart as suggested here: https://wiki.tiltedphoques.com/tilted-online/guides/troubleshooting/the-strui-doesnt-appear-when-i-press-right-ctrl-or-f2
